[2025-03-15T14:01:44.603+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T14:01:44.618+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T14:01:43.317585+00:00 [queued]>
[2025-03-15T14:01:44.627+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T14:01:43.317585+00:00 [queued]>
[2025-03-15T14:01:44.629+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-15T14:01:44.643+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): api> on 2025-03-15 14:01:43.317585+00:00
[2025-03-15T14:01:44.648+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1193) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T14:01:44.650+0000] {standard_task_runner.py:72} INFO - Started process 1195 to run task
[2025-03-15T14:01:44.650+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_wether_dag', 'api', 'manual__2025-03-15T14:01:43.317585+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_wether.py', '--cfg-path', '/tmp/tmp_qbaeuw2']
[2025-03-15T14:01:44.653+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask api
[2025-03-15T14:01:44.698+0000] {task_command.py:467} INFO - Running <TaskInstance: api_wether_dag.api manual__2025-03-15T14:01:43.317585+00:00 [running]> on host 157b2d7c5f40
[2025-03-15T14:01:44.903+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_wether_dag' AIRFLOW_CTX_TASK_ID='api' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T14:01:43.317585+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T14:01:43.317585+00:00'
[2025-03-15T14:01:44.904+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-15T14:01:45.103+0000] {python.py:240} INFO - Done. Returned value was: ok
[2025-03-15T14:01:45.126+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-15T14:01:45.127+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_wether_dag, task_id=api, run_id=manual__2025-03-15T14:01:43.317585+00:00, execution_date=20250315T140143, start_date=20250315T140144, end_date=20250315T140145
[2025-03-15T14:01:45.189+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T14:01:45.211+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-15T14:01:45.214+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
